thirdeye/thirdeye-frontend/app/pods/manage/alerts/controller.js

```diff
@@ -44,11 +65,17 @@ export default Ember.Controller.extend({
     // Handles alert selection from type ahead
     onAlertChange(alerts) {
       this.get('selectedAlerts').pushObject(alerts);
+      this.set('resultsActive', true);
     },
 
     // Handles UI mode change
     onModeChange(mode) {
       this.set('selectedSearchMode', mode);
     },
+
+    removeAll() {
+      $('.te-search-results').remove();
```

Comment:
Do we have to use jquery here? wouln't we achieve the same result by wrapping the `'.te-search-results'` section with an `{{#if resultsActive}}` ?